t/rest_server/workflows/contents/workflows/deploy-rest-server.yaml

```diff
@@ -41,6 +41,9 @@ jobs:
       id-token: 'write'
     name: 'Release Image'
     runs-on: 'ubuntu-latest'
+    defaults:
+      run:
+        working-directory: './code/rest_server'
```

Comment:
Note: this workflow expects the opinionated folder structure, not sure how to work around this if --dest is omitted.